lib/macro/internal/Cargo.toml

```
@@ -20,6 +20,6 @@ default-features = false
 features = ["alloc"]
 
 [dependencies.syn]
-version = "1"
+version = "2"
```

I'll run the workflow, but I suspect this might break MSRV. I think we need to support both 1 and 2.
```suggestion
version = ">= 1, < 3"
```